src/window.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio [WSL : Ubuntu]
+** File description:
+** window
+*/
+#include "Window.hpp"
+
+
+indie::Window::Window(int screenWidth, int screenHeight, const std::string & name ) {
+    char* title = const_cast<char*>(name.c_str());
```

Comment:
Why do you need this cast ?

Reply:
because init window needs a const char * and that was the answer i found to convert std::string to char * on google ^^
is there another possibility ?